apps/auction/templates/my_active_bids.jade

```diff
@@ -3,27 +3,43 @@ if myActiveBids && myActiveBids.length
   table
     tbody
       for bid in myActiveBids
-        tr
-          td.auction-mab-artist-artwork-container
-            a(
-              href=bid.sale_artwork.artwork.href
-            )
-              img.auction-mab-img( src=bid.sale_artwork.artwork.image.url )
-              .auction-mab-artist-artwork
-                h3= bid.sale_artwork.artwork.artist.name
-                .auction-mab-title
-                  em= bid.sale_artwork.artwork.title
-                  | ,&nbsp;
-                  = bid.sale_artwork.artwork.date
-          td.auction-mab-lot-number Lot #{bid.sale_artwork.lot_number}
-          td.auction-mab-current-bid
-            strong Current Bid: <em>#{bid.sale_artwork.highest_bid.display}</em>
-          td.auction-mab-bid-num (#{bid.sale_artwork.counts.bidder_positions} Bids)
-          if bid.is_highest_bidder
-            td.auction-mab-warning.is-winning: p Highest Bid
-          else
-            td.auction-mab-warning: p Outbid
-          td
-            a.avant-garde-button-black.auction-mab-bid-button(
-              href=bid.sale_artwork.artwork.href
-            ) Bid
+        //- defend against empty bids
+        if bid.sale_artwork
```

Comment:
Hopefully this is fixed with the recent gravity change, but thanks for adding the defensive check here!